apps/end-tenancy/models/upload.js

```diff
@@ -2,69 +2,70 @@
 'use strict';
 
 const url = require('url');
+const FormData = require('form-data');
 const Model = require('hof').model;
 const config = require('../../../config');
 const debug = require('debug')('upload-model');
 
 module.exports = class UploadModel extends Model {
-  save() {
-    return new Promise((resolve, reject) => {
+  async save() {
+    try {
       const attributes = {
         url: config.upload.hostname
       };
       const reqConf = url.parse(this.url(attributes));
-      reqConf.formData = {
-        document: {
-          value: this.get('data'),
-          options: {
-            filename: this.get('name'),
-            contentType: this.get('mimetype')
-          }
-        }
-      };
+      const formData = new FormData();
+      formData.append('document', this.get('data'), {
+        filename: this.get('name'),
+        contentType: this.get('mimetype')
+      });
+      reqConf.data = formData;
       reqConf.method = 'POST';
+      reqConf.headers = {
+        ...formData.getHeaders()
+      };
       debug('SAVE PDF DATA', reqConf);
-      this.request(reqConf, (err, data) => {
-        if (err) {
-          return reject(err);
-        }
-        debug('RESPONSE FROM FILE VAULT SAVE', err, data);
-        return resolve(data);
-      });
-    });
-  }
+      const response = await this.request(reqConf);
+      debug('RESPONSE FROM FILE VAULT SAVE', response);
```

Comment:
not a suggestion just more of a question, do you think this should be info rather than debug in the logs?

Reply:
Yeah, looking at the commit history, debug was used to replace console.logs so replacing this with the hof logger at the info level would be more robust. I can create a ticket for looking at the projects that need to have the hof logger implemented

Reply:
Was a quick change, so I have replaced debug and console logs with the hof logger  in this repo now.

Reply:
nice one